corehq/apps/locations/permissions.py

```diff
@@ -103,11 +104,16 @@
 
 
 def locations_access_required(view_fn):
-    """
-    Decorator controlling domain-level access to locations.
-    """
-    return login_and_domain_required(
+    """Decorator controlling domain-level access to locations features."""
+    return (login_and_domain_required(
         requires_privilege_raise404(privileges.LOCATIONS)(view_fn)
+    ))
+
+
+def require_can_edit_locations(view_fn):
```

Comment:
I agree it's a little confusing.  `locations_access_required` checks whether the domain can use locations (is it available on their plan?).  `require_can_edit_locations` checks whether the user has permission to create, update, and delete locations.  I pulled that name off of `require_can_edit_commcare_users`, `require_can_edit_web_users`, and a few others.

The former isn't really view _or_ edit access - it doesn't refer to individual locations, but to the section of the site concerning locations.